test/utils/mapbox-transform.js

```diff
@@ -20,10 +20,25 @@
 
 // NOTE: Transform is not a public API so we should be careful to always lock
 // down mapbox-gl to a specific major, minor, and patch version.
-import Transform from 'mapbox-gl/js/geo/transform';
-import {LngLat, Point} from 'mapbox-gl';
+import {Map, LngLat, Point} from './mapbox';
 
-export {LngLat, Point, Transform} from 'mapbox-gl';
+let Transform;
+/* eslint-disable */
+// Hack: mapbox-gl does not expose Transform class
+try {
+  Map.prototype = Object.assign({}, Map.prototype, {
+    _setupContainer: function() {
+      Transform = this.transform.constructor;
+    }
+  });
+  new Map();
+
+} catch (err) {
+  // Ignore
```

Comment:
Map constructor throws error without WebGL, but we just need the Transform class constructor here.

Reply:
The way this used to work in react-map-gl is that mapbox under node looked for the presence of headless-gl and automatically loaded it if available. So just installing `gl` and a few more modules (`sinon` and maybe one other) used to make mapbox run under node.